Formula/g/gql.rb

```diff
@@ -0,0 +1,23 @@
+class Gql < Formula
+  desc "Git Query language is a SQL like language to perform queries on .git files"
+  homepage "https://github.com/AmrDeveloper/GQL"
+  url "https://github.com/AmrDeveloper/GQL/archive/refs/tags/0.25.0.tar.gz"
+  sha256 "7eb75c9bb49f8b51524155b9c5d64294ac2bcf2b00812e3c39c1dd7e45aaf96e"
+  license "MIT"
+  head "https://github.com/AmrDeveloper/GQL.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  def install
+    require "fileutils"
```

Comment:
```suggestion
  def install
```
I don't think you need this.

Reply:
Ok,

There is already a formula with name gitql 

https://formulae.brew.sh/formula/gitql

Reply:
Done